index.html

```diff
@@ -1,13 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>"ALL about me"</title>
+    <meta name="Farida" content="HTML"/>
+  </head>
+  <body>
 <h1>Farida Hyderi</h1>
+    <section id="About">
 <h2>About</h2>
 <p>My name is Farida I am very entusiastic to learn new programming langusges
 also very much interested in testing </p>
+    </section>
+
+    <section id = "Experience">
 <h2>Experience</h2>
-Cognizant Agile Methodology from Forage
-Accenture Project management Forage
-Mastercard - Cybersecurity Job Simulation from Forage
-Worked on testing projects
+<li>Cognizant Agile Methodology from Forage </li>
+<li>Accenture Project management Forage </li>
+<li>Mastercard - Cybersecurity Job Simulation from Forage</li>
+<li>Worked on testing projects</li>
```

Comment:
For the bullet points on this list to work, you'll need to wrap the entire thing in either `<ul>` for an unordered list. Take a look at [this reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).

Reply:
This fix works just fine. Nice job.